packages/edit-site/src/components/template-details/index.js

```diff
@@ -32,10 +32,8 @@ export default function TemplateDetails( { template, onClose } ) {
 
 	// TODO: We should update this to filter by template part's areas as well.
 	const browseAllLinkProps = useLink( {
-		canvas: 'view',
-		postType: template.type,
-		postId: undefined,
 		path: '/' + template.type + '/all',
+		canvas: undefined,
```

Comment:
Do we still need this, since you changed the `history.push`?

Reply:
probably not.